hardware/tftp/Android.mk

```diff
@@ -54,6 +57,13 @@ SONY_SYMLINKS += \
         /data/vendor/tombstones/rfs/slpi:$(TARGET_COPY_OUT_VENDOR)/rfs/$(tgtsoc)/slpi/ramdumps \
     )
 
+ifeq ($(TARGET_USES_ESOC),true)
+SONY_SYMLINKS += \
+    $(foreach tgtsoc,$(target_soc), \
+        /data/vendor/tombstones/rfs/tn:$(TARGET_COPY_OUT_VENDOR)/rfs/$(tgtsoc)/tn/ramdumps \
+    )
+endif
```

Comment:
At this point we might as well wrap these in a:

```mk
SONY_SYMLINKS += \
    $(foreach tgtsoc,$(target_soc), \
        $(foreach prefix,$(target_prefixes), \
            /data/vendor/tombstones/rfs/$(prefix):$(TARGET_COPY_OUT_VENDOR)/rfs/$(tgtsoc)/$(prefix)/ramdumps
        ) \
    )
```
And accept that the one for `adsp` and `mpss` have a different "from" path. Can't hurt to have those symlinks /methinks, the pattern is generic enough and already repeating for `cdsp`/`slpi`/`tn`.

Reply:
Maybe. If you want, do a followup after this PR gets in.

Reply:
Now would be nice though...